azurelinuxagent/ga/monitor.py

```diff
@@ -418,10 +418,36 @@ def send_cgroup_telemetry(self):
 
         if datetime.datetime.utcnow() >= (self.last_telemetry_heartbeat + MonitorHandler.CGROUP_TELEMETRY_PERIOD):
             try:
-                for cgroup_name, metrics in CGroupsTelemetry.collect_all_tracked().items():
+                metric_reported, metric_threshold = CGroupsTelemetry.collect_all_tracked()
+                for cgroup_name, metrics in metric_reported.items():
+                    thresholds = metric_threshold[cgroup_name]
+
                     for metric_group, metric_name, value in metrics:
                         if value > 0:
                             report_metric(metric_group, metric_name, cgroup_name, value)
+
+                        if metric_group == "Memory":
+                            if value > thresholds["memory"]:
```

Comment:
```suggestion
                            if value >= thresholds["memory"]:
```

CGroups should whack memory usage down until it meets the defined limit; if cgroups is working correctly, you won't seen an "exceeds" value, but you will see an "equals" value.